tools/deprecate_version/deprecate_version.py

```diff
@@ -117,10 +117,16 @@ def CreateIssues(access_token, runtime_and_pr):
       try:
         repo.create_issue(title, body=body, assignees=[user.login], labels=labels)
       except github.GithubException as e:
-        print(('GithubException while creating issue. This is typically because'
-               ' a user is not a member of envoyproxy org. Check that %s is in '
-               'the org.') % user.login)
-        raise
+        try:
+          body += '\ncc @' + user.login
+          repo.create_issue(title, body=body, labels=labels)
+          print(('unable to assign issue %s to %s. Add them to the Envoy proxy org'
+                 'and assign it their way.') % (title, user.login))
+        except github.GithubException as e:
+          print(('GithubException while creating issue. This is typically because'
```

Comment:
nit: do you want to change the printed message given that the user not being in the org is handled above?